triton-server/src/notification.ts

```diff
@@ -75,15 +75,15 @@ export const sendDatasetValidationStatusUpdateEmail = async () => {
                     },
                 )
             }
-            let body =
-                "A run has been validated: <br/>" +
+            const body =
+                "<b>A run has been validated:</b> <br/>" +
                 formattedData.map(
                     (dataset: ExtractedValidatedNotificationData) => {
                         return `<br/><br/>
-                            -   Run Name: ${dataset.projectAndRunInfo.run_name} <br/>
-                            -   Validated by: ${dataset.projectAndRunInfo.validated_by} <br/>
-                            -   Project: ${dataset.projectAndRunInfo.project_name}  ${dataset.projectAndRunInfo.project_id ?? ""} <br/>
-                            - Dataset/lane ${dataset.projectAndRunInfo.lane_number} status ${getValidationFlagLabel(dataset.projectAndRunInfo.validation_status)} <br/>
+                            -   <b>Run Name:</b> ${dataset.projectAndRunInfo.run_name} <br/>
+                            -   <b>Validated by:</b> ${dataset.projectAndRunInfo.validated_by} <br/>
+                            -   <b>Project:</b> ${dataset.projectAndRunInfo.project_name}  ${dataset.projectAndRunInfo.project_id ?? ""} <br/>
+                            - <b>Dataset/lane ${dataset.projectAndRunInfo.lane_number} status</b> ${getValidationFlagLabel(dataset.projectAndRunInfo.validation_status)} <br/>
                                 ${dataset.basicCommentUserInfo?.comment != undefined ? "- Comments: " + dataset.basicCommentUserInfo?.comment + "<br/>" : "No comments <br/>"}
                                 ${dataset.basicCommentUserInfo?.comment != undefined ? "- Comments left by: " + dataset.basicCommentUserInfo?.name + "<br/>" : ""}
```

Comment:
`No comments` is missing a dash.

Reply:
it never had a dash to begin with, personally i dont think it should have any because theres nothing to signal with the message